_episodes/05-counting-mining.md

```diff
@@ -674,7 +674,7 @@ Pair up with your neighbor and work on these exercises:
 > > {: .bash}
 > >
 > > If you came up with something more advanced, perhaps including word boundaries,
-> > please share your result on the Etherpad and give yourself a pat on the shoulder.
+> > please share your result in the collaborative document and give yourself a pat on the shoulder.
```

Comment:
@katrinleinweber is collaborative document the preferred term? That makes sense and I wonder if we reference Etherpad in other places. Thanks for pointing this out. This is just a comment, I agree with the change.

Reply:
Not in the text anywhere, just in [`check_etherpad()`](https://github.com/LibraryCarpentry/lc-shell/blob/0e11b7e110b874264edf2d247646de2f8707c88d/bin/check-workshop#L237) which is [already updated in the `styles` repo](https://github.com/carpentries/styles/blob/6d015fe8f75ad7435a22aa8491dcfc66eed3ebb6/bin/workshop_check.py#L237), so I'm leaving that alone here.